runtime/doc/map.txt

```diff
@@ -1431,6 +1431,128 @@ Possible values are (second column is the short name used in listing):
     -addr=other		  ?	other kind of range
 
 
+Incremental preview ~
+                                                  *:command-preview* {nvim-api}
+It's possible to allow commands to be incrementally previewed when
+'inccommand' is enabled by using the preview command attribute. The preview
+attribute may only be used through the API. (see |nvim_create_user_command()|)
+
+The value of the preview attribute must be either a boolean value, the name of
+a Vim script function or a Lua function callback. Here are some examples: >
+
+	-- setting preview to a boolean value
+	call nvim_create_user_command('foo', 'call Foo()',
+		\ { 'preview': v:true })
+
+	-- setting preview to a VimScript function
+	function! PreviewFunc()
+	    return v:true
+	endfunction
+
+	call nvim_create_user_command('bar', 'call Bar()',
+		\ { 'preview': 'PreviewFunc' })
+
+	-- setting preview to a Lua function callback
+	lua << EOF
+	local function PreviewFunc()
+	    return true
+	end
+
+	vim.api.nvim_create_user_command('baz', 'call Baz()',
+	                                 { preview = PreviewFunc })
+	EOF
+<
+
+If preview is a boolean value and its value is |TRUE|, preview is always
```

Comment:
There's more examples but I'll just mention this one and hope that the pattern is applied generally: don't use lots of words to say something that can be said clearly with fewer words. "is a boolea and is TRUE" is totally redundant. Just say "is TRUE".
```suggestion
If preview is |TRUE|, preview is always
```